scripts/GSAS-II/install_gsas_vetted.bat

```diff
@@ -0,0 +1,3 @@
+@echo off
+@set VETTED_REVISION_NUMBER=3216
```

Comment:
Quick question. Do the scripts handle the upgrade when we change this number and do they effectively just update the install or does it have to be wiped out?

Reply:
It doesn't do any updating - if this number changes we want to just delete the installation and start again